### DISCUSSION TITLE
Looking for help from an Ubuntu packaging expert

### DISCUSSION THREAD

**Discussion by Olga-Yakovleva** [RHVoice/RHVoice#788] [General]
Hi everyone!

I am currently working as part of a team developing voices for several languages. The first of them is going to be released soon here on Github together with some updates to the core it will need.

The project has certain operating system compatibility requirements, including Linux support. Specifically, packages for Ubuntu 20.04 and later versions are required. I'm running Ubuntu as a user these days, but I know nothing about packaging for it and can't find the time to learn. This is why we are urgently looking for a developer who could help. Of course, there will be a compensation.

What we are looking for: guidance on how to update the sources of the existing packages to work with the given version of the core and additional languages and voices. Instructions on how to build the new packages. Possibly building binary packages for us if doing it ourselves proves to be impractical.

If anyone is interested in helping with this, please contact me privately at olga@rhvoice.org.

**Comment by zstanecic** [comment]
Hi Olga!

Let me express my opinion on this matter:  

As per my knowledge, several people built and packaged the RHVoice in the respective launchpad and or some other type of repository.

RHVoice is even packaged in the official ubuntu repos starting from ubuntu 22.03.

Why just not ask ubuntu developers to maintain the updates?

 

 

From: Olga Yakovleva ***@***.***> 
Sent: Wednesday, September 13, 2023 1:31 PM
To: RHVoice/RHVoice ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [RHVoice/RHVoice] Looking for help from an Ubuntu packaging expert (Discussion #788)

 

Hi everyone!

I am currently working as part of a team developing voices for several languages. The first of them is going to be released soon here on Github together with some updates to the core it will need.

The project has certain operating system compatibility requirements, including Linux support. Specifically, packages for Ubuntu 20.04 and later versions are required. I'm running Ubuntu as a user these days, but I know nothing about packaging for it and can't find the time to learn. This is why we are urgently looking for a developer who could help. Of course, there will be a compensation.

What we are looking for: guidance on how to update the sources of the existing packages to work with the given version of the core and additional languages and voices. Instructions on how to build the new packages. Possibly building binary packages for us if doing it ourselves proves to be impractical.

If anyone is interested in helping with this, please contact me privately at ***@***.*** ***@***.***> .

**Comment by Olga-Yakovleva** [comment]
At the moment I'm just looking for a quick fix to satisfy the contract requirements. It's a separate question what a long-term solution must be. I don't know what the Ubuntu packaging and updates workflow looks like. Ideally, I'd prefer a universal solution like snaps. But right now we've got deadlines to make.
